scene/main/viewport.cpp

```diff
@@ -773,10 +773,53 @@ bool Viewport::is_audio_listener_2d() const {
 	return audio_listener_2d;
 }
 
+void Viewport::enable_canvas_transform_override(bool p_enable) {
+#ifdef TOOLS_ENABLED
```

Comment:
Make sure this runs without tools too, else it won't work when running on something like Android